Makefile

```
@@ -127,7 +127,7 @@ deploy-app: ## Deploys the app to PaaS
 .PHONY: deploy-db-migration
 deploy-db-migration: ## Deploys the db migration app
 	$(if ${APPLICATION_NAME},,$(error Must specify APPLICATION_NAME))
-	cf push ${APPLICATION_NAME}-db-migration -f <(make -s -C ${CURDIR} generate-manifest) -o digitalmarketplace/${APPLICATION_NAME}:${RELEASE_NAME} --no-route --health-check-type none --health-check-http-endpoint none -i 1 -m 128M -c 'sleep 2h'
+	cf push ${APPLICATION_NAME}-db-migration -f <(APPLICATION_NAME=db-migration make -s -C ${CURDIR} generate-manifest) -o digitalmarketplace/${APPLICATION_NAME}:${RELEASE_NAME} --no-route
```

I'm a bit confused about using `APPLICATION_NAME=db-migration` and `${APPLICATION_NAME}:${RELEASE_NAME}` (which presumably has `APPLICATION_NAME=api` to get the right Docker image) in the same line...?

---

This might be clearer/less prone to failure

```suggestion
	cf push ${APPLICATION_NAME}-db-migration -f <(make -s -C ${CURDIR} generate-manifest APPLICATION_NAME=db-migration) -o digitalmarketplace/${APPLICATION_NAME}:${RELEASE_NAME} --no-route
```

---

Although to be honest, since we're special casing the api app here, we might as well take out the outer ${APPLICATION_NAME} references and hardcode api

```suggestion
	cf push api-db-migration -f <(APPLICATION_NAME=db-migration make -s -C ${CURDIR} generate-manifest) -o digitalmarketplace/api:${RELEASE_NAME} --no-route
```